web/src/features/modals/AnnouncementModal.tsx

```diff
@@ -0,0 +1,83 @@
+import { SourceColorAnnouncementImage } from 'icons/announcementSourceColorImage';
+import { useAtom } from 'jotai';
+import { atomWithStorage } from 'jotai/utils';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { HiXMark } from 'react-icons/hi2';
+import { hasOnboardingBeenSeenAtom } from 'utils/state/atoms';
+
+const hasAnnouncementBeenSeenAtom = atomWithStorage(
+  'announcementSeen',
+  localStorage.getItem('announcementSeen') ?? false
+);
+
+const handleTierGaterClick = (event: React.MouseEvent<HTMLDivElement>) => {
+  event.stopPropagation();
+  event.preventDefault();
+};
+
+export default function AnnouncementModal() {
+  const { t } = useTranslation();
+  const [hasAnnouncementBeenSeen, setHasAnnouncementBeenSeen] = useAtom(
+    hasAnnouncementBeenSeenAtom
+  );
+  const [hasOnboardingBeenSeen] = useAtom(hasOnboardingBeenSeenAtom);
+  const [hasOnboardingBeenSeenInSession, setHasOnboardingBeenSeenInSession] =
+    useState(false);
+  const isExpired = new Date() > new Date('2024-06-30');
```

Comment:
Before we actually get this out and mobile builds reviewed and released it might be a bit short time-span, so I suggest we extend it like this
```suggestion
  const isExpired = new Date() > new Date('2024-07-15');
```